.circleci/config.yml

```diff
@@ -162,95 +164,94 @@ jobs:
 workflows:
   check-and-deploy-development:
     jobs:
-      - check-code-formatting:
-          context: api-nuget-token-context
+      - check-code-formatting
+      - build-and-test
+      - assume-role-development:
+          context: 
+            - api-assume-role-housing-development-context
+            - SonarCloud
```

Comment:
The `assume-role-...` command _(ln26)_ does not depend on `SonarCloud` context secrets.